packages/vite/src/node/plugins/css.ts

```diff
@@ -1803,6 +1883,23 @@ async function minifyCSS(
       ...resolveMinifyCssEsbuildOptions(config.esbuild || {}),
```

Comment:
```suggestion
      filename,
      ...resolveMinifyCssEsbuildOptions(config.esbuild || {}),
```

Reply:
Accoring to type definition and [docs](https://esbuild.github.io/api/#sourcefile), I set the the key to be `sourcefile`.